qemu/tests/qemu_output.py

```diff
@@ -0,0 +1,42 @@
+import re
+import logging
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verify qemu has no error outputs at different stages in the guest life cycle.
+
+    1) Launch a guest.
+    2) Check qemu outputs have not error messages.
+    3) Reboot guest and check qemu outputs.
```

Comment:
I check the case steps, here I need to start the L2 guest, and restart the check.
Here I did not see the related operations of L2 guest.

Reply:
Add more operations is in order to cover more scenarios.

Reply:
Sorry, I am a little confused. So should I use another script to boot L2 guest? Or does this script complete the L2 guest startup procedure?

Reply:
This is the script running in L1 via ansible-playbook, we only need to develop nested test cases like normal test cases.

Reply:
Thanks for the feedback,  I understand.